burgle.lic

```diff
@@ -136,13 +390,17 @@ class Burgle
       handheld = held_item(item)
       drop_item(held_item(item)) if handheld
       return false
+    else
+      drop_item(item)
+      return false
     end
 
     false
   end
 
+  #ripped out of steal.lic
   def drop_item(item)
-    case bput("drop my #{item}", 'You drop', 'would damage it', 'smashing it to bits', 'Something appears different about', 'What were you')
+    case bput("drop my #{item}", 'You drop', 'You wince', 'would damage it', 'smashing it to bits', 'Something appears different about', 'What were you')
```

Comment:
@vtcifer Do you recall the game message that led to adding `You wince` to the list of match strings?

Per https://github.com/rpherbig/dr-scripts/pull/4959/files#r634241388 and https://github.com/rpherbig/dr-scripts/pull/4161/files#r382959034, I'm working on updates and tests for `common-items` so that `burgle` and `stealing` can switch to `DRCI.dispose_trash(item)`. On that journey, I want to make sure that I've got all the use cases handled.

Thanks

cc @rpherbig

Reply:
It's from when you drop something that breaks or gets damaged.
Instruments, gaethzen, telescopes.  One of those.